docs/deprecations.md

```diff
@@ -149,3 +149,47 @@ Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be re
 ## `bin/serverless`
 
 Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2.0.0
+
+## To Disable specific Deprecations
```

Comment:
I think it would be better to move that section to the top of the document, right after `# Serverless Framework Deprecations`. I would also call this section: `How to disable specific deprecations`.

Reply:
I believe It would be better if it

# How to disable specific deprecation logs